simtools/visualization/legend_handlers.py

```diff
@@ -15,55 +20,72 @@
     "MeanRadiusOuterEdgeObject",
     "OffPixelObject",
     "PixelObject",
+    "SCTHandler",
+    "SCTObject",
     "SSTHandler",
     "SSTObject",
     "SquareEdgePixelHandler",
     "SquareOffPixelHandler",
     "SquarePixelHandler",
+    "TelescopeHandler",
 ]
 
 
+class TelescopeHandler(object):
+    """
+    Telescope handler that centralizes the telescope information. Individual telescopes handlers
+    inherit from this class.
+    """
+
+    def __init__(self):
+        io_handler = IOHandler()
+        corsika_parameters_file = io_handler.get_input_data_file(
+            "parameters", "corsika_parameters.yml"
+        )
+        corsika_info = CorsikaConfig.load_corsika_parameters_file(corsika_parameters_file)
+
+        self.radius_dict = {}
+        self.colors_dict = {
+            lst: "darkorange",
+            mst: "dodgerblue",
+            sct: "black",
+            sst: "lightsteelblue",
```

Comment:
I indicated before that I don't like having telescope types hardwired in too many places. Leave it for now, but it is not nice.

Reply:
the telescopes are not hardwired here, these are parameters which define the strings we use for the telescopes. the definition is centralized in `names.py`. everywhere else we import and use these variables. if at some point we want to change, we do it in a single place. If at some point we want to add a different kind of telescope we would indeed need to add it to the definitions to `names.py` and add the variables throughout the code (DB and so on..). I dont see a way to cover all kinds of telescopes other than this. Do you have an idea how to improve this?

By the way, I centralized the colors now as well. Now they are defined only in TelescopeHandler for all telescopes.